static/app/utils/discover/fieldRenderers.tsx

```diff
@@ -433,37 +436,95 @@ const SPECIAL_FIELDS: SpecialFields = {
   },
   assignee: {
     sortField: 'assignee.name',
-    renderFunc: data => {
-      const assignedTo: Actor = {
-        type: data['assignee.type'],
-        id: data['assignee.id'],
-        name: data['assignee.name'],
-        email: data['assignee.email'],
-      };
+    renderFunc: (_, {issueData}) => assigneeRenderer(issueData),
+  },
+};
 
-      return (
-        <ActorContainer>
-          {assignedTo.type && assignedTo.id && assignedTo.name ? (
-            <ActorAvatar
-              actor={assignedTo}
-              size={28}
-              tooltip={t(
+const assigneeRenderer = (issueData?: IssueTableData) => {
+  const {assignedTo, suggestedAssignees} = issueData
+    ? issueData
+    : {assignedTo: undefined, suggestedAssignees: undefined};
+  const suggestedReasons: Record<SuggestedOwnerReason, React.ReactNode> = {
+    suspectCommit: tct('Based on [commit:commit data]', {
+      commit: (
+        <TooltipSubExternalLink href="https://docs.sentry.io/product/sentry-basics/integrate-frontend/configure-scms/" />
+      ),
+    }),
+    ownershipRule: t('Matching Issue Owners Rule'),
+  };
+  const assignedToSuggestion = suggestedAssignees?.find(
+    actor => actor.id === assignedTo?.id
+  );
+  return (
+    <ActorContainer>
+      {assignedTo ? (
+        <ActorAvatar
+          actor={assignedTo}
+          size={28}
+          tooltip={
+            <TooltipWrapper>
+              {t(
                 `Assigned to ${
                   assignedTo.type === 'team' ? `#${assignedTo.name}` : assignedTo.name
                 }`
               )}
-            />
-          ) : (
-            <Tooltip isHoverable skipWrapper title={<div>{t('Unassigned')}</div>}>
-              <IconUser size="20px" color="gray400" />
-            </Tooltip>
-          )}
-        </ActorContainer>
-      );
-    },
-  },
+              {assignedToSuggestion && (
+                <TooltipSubtext>
+                  {suggestedReasons[assignedToSuggestion.suggestedReason]}
+                </TooltipSubtext>
+              )}
+            </TooltipWrapper>
+          }
+        />
+      ) : suggestedAssignees && suggestedAssignees.length > 0 ? (
+        <SuggestedAvatarStack
+          size={24}
+          owners={suggestedAssignees}
+          tooltipOptions={{isHoverable: true}}
+          tooltip={
+            <TooltipWrapper>
+              <div>
+                {tct('Suggestion: [name]', {
+                  name:
+                    suggestedAssignees[0].type === 'team'
+                      ? `#${suggestedAssignees[0].name}`
+                      : suggestedAssignees[0].name,
+                })}
+                {suggestedAssignees.length > 1 &&
+                  tn(' + %s other', ' + %s others', suggestedAssignees.length - 1)}
```

Comment:
We should combine this into a single `tn` if possible, the ordering of this phrase may be different for different languages.

Reply:
Removed this change, it is no longer necessary due to updates to reuse AssigneeSelector!